### PR TITLE
Hcap 412 add withdraw to site details view

### PR DESCRIPTION
**Pull request by dbayly-freshworks**
Added missing AC for 412. 

Health Authorities can now withdraw candidates from the Site details page

Screen Shot 2021-06-24 at 2 47 55 PM
Screen Shot 2021-06-24 at 2 47 47 PM

### CODE REVIEW

**Review by fwpushan — APPROVED** [review]
LGTM

File: client/src/pages/private/SiteParticipantsTable.js [hunk 1]
```diff
@@ -31,6 +49,48 @@ export default ({ siteId }) => {
 
   const sort = (array) => _orderBy(array, [orderBy, 'operatorName'], [order]);
 
+  const forceReload = async () => {
```
Comment (Zenen-freshworks) [inline client/src/pages/private/SiteParticipantsTable.js:52]: is this code taken from the useEffect on line 94? I think we would be better off encapsulating it.

**Review by Zenen-freshworks — COMMENTED** [review]
Just one thing I'd like to see changed, otherwise it's looking like a solid PR

File: client/src/pages/private/SiteParticipantsTable.js (hunk 1 above)
Comment (dbayly-freshworks) [reply client/src/pages/private/SiteParticipantsTable.js:52]: I know this is an anti pattern, but we're kinda forced into it. We need to be able to reload the form after we submit. 
If we just have it declared within the useEffect, then we can't call it elsewhere. If we declare it outside, it becomes a dependancy of the useEffect and breaks. 
I don't know if there is a more elegant way to handle this issue. 

We use this pattern exactly in the participants table.

Comment (Zenen-freshworks) [reply client/src/pages/private/SiteParticipantsTable.js:52]: Ah, fair enough! I forgot about the whole dependency thing.

**Pull request merged**